packages/slice-machine/lib/models/common/widgets/StructuredText/Mock/Form.js

```
@@ -23,10 +25,12 @@ const HandlePatternTypes = ({
 }) => {
   const { theme } = useThemeUI();
 
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-argument
```

Normally this is not something that we do.
Even though I totally understand that it's your role to fixe those issues, it's still painful to see new errors being ignored.

---

code changes and still rely on old code that is not typed, it has to happen unless we get rid of JS